### PR TITLE
data:Parthmandale

### PR DESCRIPTION
**Pull request by Parthmandale**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6353"></a>

### CODE REVIEW

**Comment by Parthmandale** [comment]
Anyone pls tell what went wrong?

File: data/Parthmandale.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Parth Mandale",
+  "type": "personal",
+  "bio": "Passionate Techie who loves creating product. **Web3.0** and **Open Source is💚**"
+  "avatar": "https://github.com/Parthmandale.png"
```
Comment (kumarsonsoff3) [inline data/Parthmandale.json:5]: This `avatar` key is no longer needed. Please remove this drom your `json`.

```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Comment by Parthmandale** [comment]
Ohk I removed it...pls review it now

**Review by Arindam200 — APPROVED** [review]

File: data/Parthmandale.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Parth Mandale",
+  "type": "personal",
+  "bio": "Passionate Techie who loves creating product. **Web3.0** and **Open Source is💚**"
```
Comment (SaraJaoude) [inline data/Parthmandale.json:4]:
```suggestion
  "bio": "Passionate Techie who loves creating product. **Web3.0** and **Open Source is💚**",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**